core/app/models/spree/order_updater.rb

```diff
@@ -77,15 +77,14 @@ def update_order_total
 
     def update_adjustment_total
       recalculate_adjustments
-      order.adjustment_total = line_items.sum(:adjustment_total) +
-      shipments.sum(:adjustment_total) +
-      adjustments.eligible.sum(:amount)
-      order.included_tax_total = line_items.sum(:included_tax_total) + shipments.sum(:included_tax_total)
-      order.additional_tax_total = line_items.sum(:additional_tax_total) + shipments.sum(:additional_tax_total)
 
-      order.promo_total = line_items.sum(:promo_total) +
-      shipments.sum(:promo_total) +
-      adjustments.promotion.eligible.sum(:amount)
+      all_items = line_items + shipments
+
+      order.adjustment_total = all_items.sum(&:adjustment_total) + adjustments.eligible.sum(:amount)
+      order.included_tax_total = all_items.sum(&:included_tax_total)
+      order.additional_tax_total = all_items.sum(&:additional_tax_total)
+
+      order.promo_total = all_items.sum(&:promo_total) + adjustments.promotion.eligible.sum(:amount)
 
       update_order_total
     end
```

Comment:
:+1: I really want to do this. I tried it and it caused a few errors, so I'd like to do that as a separate follow-up to this PR